### PR TITLE
dag: Fixes to tests

### PR DESCRIPTION
**Pull request by jdu**
# Description

- Fixes some tests and some issues in the MSF scraper

## Type of change

Please delete options that are not relevant.

- [x] :bug: Bug fix 

# How Has This Been Tested?

`make docker-test`

# Checklist:

- [x] My code follows the style guidelines of this project (pep8 AND pyflakes)
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] If needed, I changed related parts of the documentation
- [x] I included tests in my PR
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] If my PR aims to fix an issue, I referenced it using `#(issue)`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -123,7 +123,7 @@ update-requirements-txt:
 
 .PHONY: test
 test: virtualenv
-	$(VIRTUALENV)/bin/pytest ./reach
+	$(VIRTUALENV)/bin/pytest -s ./reach
```
Comment (SamDepardieu) [inline Makefile]: What does -s do?

File: reach/scraper/tests/mock_sites/msf/1.html [hunk 2]
```diff
@@ -5,21 +5,106 @@
...
+<meta name="twitter:title" content="Activity Reports" />
+<meta name="twitter:description" content="Our International Activity Report details our field work during the preceding 12 months" />
+<meta name="twitter:image" content="https://www.msf.org.uk/sites/uk/files/styles/large/public/issues/msf148284_high_res.jpg?itok=MY6i9dCz" />
+<meta name="twitter:image:width" content="480" />
+<meta name="twitter:image:height" content="320" />
+<meta property="article:published_time" content="2016-09-22T23:28:56+00:00" />
+<meta property="article:modified_time" content="2019-12-13T12:07:57+00:00" />
+<meta itemprop="name" content="Activity Reports" />
+<meta itemprop="description" content="Our International Activity Report details our field work during the preceding 12 months" />
+<meta name="msvalidate.01" content="B6273418D0915000CF0D7D849FEAEF1B" />
```
Comment (SamDepardieu) [inline reach/scraper/tests/mock_sites/msf/1.html]: Pretty sure you can remove those meta tags

File: Makefile (hunk 1 above)
Comment (jdu) [reply Makefile]: Shows `print()` call output for debugging

File: reach/scraper/tests/mock_sites/msf/1.html (hunk 2 above)
Comment (jdu) [reply reach/scraper/tests/mock_sites/msf/1.html]: Good catch! WIll push a remove on those.

**Pull request merged**